sound/soc/codecs/max98373-sdw.c

```diff
@@ -780,6 +780,9 @@ static int max98373_sdw_set_tdm_slot(struct snd_soc_dai *dai,
 		snd_soc_component_get_drvdata(component);
 
 	/* tx_mask is ignored since it's irrelevant for I/V feedback */
+	if (tx_mask)
```

Comment:
Should we also change the comment? Now, tx_mask is not just ignored, but invalid.